pypose/module/imu_preintegrator.py

```diff
@@ -277,8 +319,8 @@ def integrate(self, init_state, dt, gyro, acc, rot:pp.SO3=None):
         incre_t = torch.cumsum(dt, dim = 1)
         incre_t = torch.cat([torch.zeros(B, 1, 1, dtype=dt.dtype, device=dt.device), incre_t], dim =1)
 
-        return {'acc':acc, 'vel':incre_v[...,1:,:], 'pos':incre_p[:,1:,:], 'rot':incre_r[:,1:,:],
-                't':incre_t[...,1:,:], 'dr': dr[:,1:,:], 'dt': dt}
+        return {'acc':a, 'vel':incre_v[...,1:,:], 'pos':incre_p[:,1:,:], 'rot':incre_r[:,1:,:],
```

Comment:
Do you need to change the doc by changing the `acc` to `a`?

Reply:
In order to show the difference between raw acceleration `acc` and "gravity-corrected" acceleration `a`. I will change the key to `a`. In our documentation, all `acc` denote the "gravity-corrected" acceleration, there are no misleading, so I think we don't need to update the doc.